README.md

```diff
@@ -26,11 +26,50 @@ The installation is basically limited to downloading the executable copy of the
 
 ### Linux environment
 
-Run the downloaded binary file after granting executables rights:
+Download the *h1-linux* file from [Releases] using wget:
 
 ```bash
-$ chmod +x h1_client.bin
-$ ./h1_client.bin
+$ wget https://github.com/hyperonecom/h1-cli/releases/latest/download/h1-linux
+```
+
+Then run the downloaded binary file after granting executables rights:
+
+```bash
+$ chmod +x h1-linux
+$ ./h1-linux
+```
+
+The application should work correctly without installing additional libraries / packages.
+
+If you intend to use the software constantly, you can install for the user or at the system level.
+
+Installation for the user can be carried out by:
+
+```bash
+$ cp h1-linux ~/.local/bin/h1
+$ h1
+```
+
+Installation at the system level can be carried out by:
+
+```bash
+$ sudo cp h1-linux /usr/local/bin/h1
+$ h1
+```
+
+### Alpine environment
+
+Download the *h1-alpine* file from [Releases] using wget:
+
+```bash
+$ wget https://github.com/hyperonecom/h1-cli/releases/latest/download/h1-alpine
+```
+
+Then run the downloaded binary file after granting executables rights:
+
+```bash
+$ chmod +x h1-alpine
+$ ./h1-alpine
 ```
 
 The application should work correctly without installing additional libraries / packages.
```

Comment:
Alpine required ```libstdc++``` according to us Dockerfile (based on Alpine). See https://github.com/hyperonecom/h1-cli/blob/master/Dockerfile .

Reply:
Added